app/services/sitemap_indexer.rb

```diff
@@ -124,4 +122,8 @@ def set_counter_callbacks
     SearchgovUrl.set_callback :create, :after, :_update_counts_after_create
     SearchgovUrl.set_callback :update, :after, :_update_counts_after_update
   end
+
+  def https_url(url)
+    UrlParser.update_scheme(url.strip, 'https')
```

Comment:
There's a bit of duplication here, as we set `https` at the model level. However, I realized that some other bit of code might eventually accidentally initialize a `SitemapIndexer` with an `http` sitemap URL, so it felt safer to enforce that in more places than might be strictly necessary.